File: app/src/main/java/com/jiangkang/ktools/requests/juejin/JuejinFragment.java
package com.jiangkang.ktools.requests.juejin;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.jiangkang.ktools.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class JuejinFragment extends Fragment {


    public JuejinFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_juejin, container, false);
    }

}

File: app/src/main/java/com/jiangkang/ktools/requests/juejin/JuejinFragment.kt
package com.jiangkang.ktools.requests.juejin


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.jiangkang.ktools.R


/**掘金APi相关
 */
class JuejinFragment : Fragment() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activity.title = "掘金"
    }


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater!!.inflate(R.layout.fragment_juejin, container, false)
    }

}// Required empty public constructor
